lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -108,6 +108,7 @@ def self.run_release_upload(params)
         owner_type = params[:owner_type]
         app_name = params[:app_name]
         destinations = params[:destinations]
+        destination_excludes = params[:destination_excludes]
```

Comment:
Hi @Hanstad 
Agree with @jpeg-ms, when we can achieve it with existing options itself.
Please check if you can remove `destination_excludes` .